.github/workflows/tox.yaml

```diff
@@ -0,0 +1,33 @@
+name: Test checkbox-provider-ce-oem with tox
+
+on:
+  push:
+  pull_request:
+    branches: [ main ]
+    paths:
+      - checkbox-provider-ce-oem/**
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python: ["3.5", "3.6", "3.8", "3.10"]
+    steps:
+      - uses: actions/checkout@v3
+      - name: install stable checkbox and checkbox-provider-ce-oem
+        run: |
+          cd ~
+          git clone --depth=1 https://github.com/hanhsuan/checkbox.git
```

Comment:
Hi @hanhsuan , 
I'm just wondering, should we clone the official checkbox repo here? or you did some changes in your repo to make things work?

Reply:
Yes, you are right. I copied the wring url.